res/ui/Tactile/flow.js

```diff
@@ -33,7 +33,7 @@ function create_steps() {
      */
```

Comment:
Update the copyright header at the top of this file to say

```
 * Copyright (C) 2018-2019 Kano Computing Ltd.
```

You should do this for any file that you change.

Reply:
ok thank you